docs/data/bestman-install.md

```diff
@@ -389,7 +389,7 @@ In the output, check that your `gumsIDMapped` is not `null`. It returns the `uid
 
 ### Verifying GridFTP
 
-Login on the node where your certificate and [OSG Client](../client/wn.md) is installed You will need to generate your proxy credentials using `grid-proxy-init` or `voms-proxy-init`.
+Login on the node where your certificate and [OSG Client](../worker-node/install-wn.md) is installed You will need to generate your proxy credentials using `grid-proxy-init` or `voms-proxy-init`.
```

Comment:
Shouldn't the test for the link be "OSG Worker Node" instead of "OSG Client"?

Reply:
I think `OSG Worker Node Client` would be the most descriptive. I'll update it.